app/templates/CreateAccount.html

```diff
@@ -3,23 +3,20 @@
     <div class="content d-flex align-items-center justify-content-center" style="height: 100vh;">
         <div class="row">
             <div class="col-md-6">
-                <div class="feedBox">
+                <div class="feedBox feedBoxSwell">
                     <h1>Create your account</h1>
-                    <button id="redirect-btn" onclick="location.href='{{LoginPage}}'" class="btn btn-sm">
-                    <span class= "fa fa-pen"></span>
-                    Already have an account?</button>
-                    <br>    
-                    <br>
-                    
-                    <p id = 'sub-header'>Please enter your account details</p>
+                    <button id="redirect-btn" onclick="location.href='{{ url_for('login') }}'" class="btn btn-sm submitButton btnSwell CreateAccountLoginSwitchBtn">
+                        <span class= "fa fa-pen"></span> Already have an account?
+                    </button>
+                    <p class = 'feedBoxSubHeader'>Please enter your account details</p>
```

Comment:
good job resolving the merge conflict, looks perfect with no breaks and correct class for padding.